scripts/pythonContractConverter.py

```diff
@@ -0,0 +1,115 @@
+# assumes all return types are strings
+def parseContract(file, funcName):
+    import re
```

Comment:
Best to put imports at the top before functions

Reply:
The reason I included the import here instead of above is because python functions which are to be converted to smart contracts need to have their imports defined within the actual function, so this preserves that stylistic behavior. In this particular instance however the import can be defined outside of the function, so I would be happy to make that change. 

Further, imports need to be specified in the C++ scope (see `runners/py/pyutil.cpp:20`) which is interesting to note, because it allows for the system designer to restrict what python libraries are available for smart contracts.

Reply:
Ok, good context, thanks for explaining the logic